### DISCUSSION TITLE
登入帳號

### DISCUSSION THREAD

**Discussion by YoutaiHuang** [hgalytoby/MyselfAnimeDownloader#30] [Q&A] [answered]
Myself登入帳號可以獲得哪些功能呢？

**Comment by hgalytoby** [comment, marked as answer]
> Myself登入帳號可以獲得哪些功能呢？

不好意思我沒收到這裡有訊息的通知，現在才看到。

很久以前看 R18 需要登入帳號才能看到，現在好像不用登入帳號了。

**Comment by YoutaiHuang** [comment]
感謝您的解惑,也謝謝您將這麽優秀的application開源
